packages/next-urql/src/with-urql-client.tsx

```diff
@@ -23,7 +23,8 @@ export function withUrqlClient(
     cacheExchange,
     ssrExchange,
     fetchExchange,
-  ]
+  ],
+  noSsr?: boolean
```

Comment:
The name isn't ideal yet but this will be used to basically indicate that the subtree is using `getInitialProps` but the wrapped page itself isn't using it.

Reply:
Gotcha – what would you think about `preferStaticRendering` or `preferStatic`?